python/cudf/cudf/tests/test_avro_reader_fastavro_integration.py

```diff
@@ -210,7 +210,7 @@ def test_can_parse_no_schema():
     assert_eq(expected, actual)
 
 
-@pytest.mark.parametrize("rows", [0, 1, 10, 100000])
+@pytest.mark.parametrize("rows", [0, 1, 10, 1000])
```

Comment:
Here I simply lowered the dataset size, but wonder if it should be larger than at least one thread block

```suggestion
@pytest.mark.parametrize("rows", [0, 1, 10, 2000])
```

Reply:
My opinion on the thread block question is that testing the correctness of the algorithm for data sizes large enough to span multiple blocks should only be a cuDF Python responsibility for 1) cases like cuIO that are much easier to test in Python than C++, or 2) functionality that does not directly map to a libcudf API for the computation. For everything else we should rely on libcudf tests.

Reply:
This is a really good point, I hadn't though of it that way. Thanks @vyasr !